convert.py

```diff
@@ -210,14 +263,67 @@ def convert_text_chunks_to_speech(ebook_chunks):
     print("All chunks converted.")
     return chunk_mp3_filepaths
 
+def merge_wavs_to_mp3(out_filename, path=".", ms_delay=100):

```

Comment:
Function documentation would be great :)

Reply:
ack